students/hirot_directory/grid_printer.py

```diff
@@ -0,0 +1,61 @@
+
+'''
+Dev: Hiro Takechi
+Subject: Grid printer
+Date: 10-15-2017
+'''
+
+plus = '+'
+minus = '-'
+space = ' '
+bar = '|'
+
+def h_line(x):
+	x = int((x-1)/2)
```

Comment:
you can use the // operator, known as "floor division" it gives an integer result, so no need for the call to int().